app/src/main/java/swati4star/createpdf/activity/MainActivity.java

```diff
@@ -127,6 +127,7 @@ protected void onDestroy() {
     protected void onResume() {
         super.onResume();
         ActionBar actionBar = getSupportActionBar();
+        actionBar.setHomeAsUpIndicator(R.drawable.ic_menu);
```

Comment:
Do we need this? The app was working fine as is before

Reply:
Please have a look at updated summary and description.